src/parser/statement.js

```diff
@@ -1188,6 +1270,24 @@ export default class StatementParser extends ExpressionParser {
     );
   }
 
+  parseClassPrivateMethod(
+    classBody: N.ClassBody,
+    method: N.ClassMethod,
+    isGenerator: boolean,
+    isAsync: boolean,
+    isConstructor: boolean,
```

Comment:
Can this ever be `true`?

Reply:
No. Not only that, it should be a syntax error; see below.

Reply:
Removed (not sure why it's not saying it's outdated).